data-collections-exercises.md

```diff
@@ -9,95 +9,113 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+planeteers[2]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers - ["Captain Planet"]
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+heroes = planeteers+ rangers
 ```
 
 Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroe.sort()
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes.shuffle
 ```
 
 #### Bonus
 
 Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes.sample
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+heroes.select{|v| v.include?('B')}
 ```
 
 ### Hashes
 
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+ninja_turtle = {
+ name:"Michelangelo",   
+ weapon:"Nunchuks",
+ radical:'true'
+}
 ```
 
 Add a key `age` to `ninja_turtle`. Set it to whatever numerical value you'd like.
 
 ```rb
-# Your answer here
+ninja_turtle = {
+ name:"Michelangelo",   
+ weapon:"Nunchuks",
+ radical:'true',
+ age:130
+}
 ```
 
 Remove the `radical` key-value pair from `ninja_turtle`.
 
 ```rb
-# Your answer here
+ninja_turtle.delete(:radicale)
 ```
 
 Add a key `pizza_toppings` to `ninja_turtle`. Set it to an array of strings (e.g., `["cheese", "pepperoni", "peppers"]`).
 
 ```rb
-# Your answer here
+ninja_turtle = {
+ name:"Michelangelo",   
+ weapon:"Nunchuks",
+ radical:'true',
+ age:130,
+ pizza_toppings:["cheese", "pepperoni", "peppers"]
+}
```

Comment:
it one way to solve this but it wouldn't be necessary to reassign the variable you can use a hash symbol to add a new property or use the merge method with `!`

```rb
ninja_turtle.merge!(pizza_toppings:["cheese", "pepperoni", "peppers"])
# or 
ninja_turtle[:pizza_toppings] = ["cheese", "pepperoni", "peppers"]
```